content/advanced/fine-grained-reactivity.mdx

```diff
@@ -155,9 +160,10 @@ function createSignal(initialValue) {
 }
 
 function createEffect(fn) {
+  const initialSubscriber = currentSubscriber;
```

Comment:
maybe "previousSubscriber" is better, like "initial" seems an absolute term, "previous" is a relative one, and I think we need a relative one here.